src/Paket.Core/Installation/InstallProcess.fs

```diff
@@ -69,12 +97,15 @@ let processContentFiles root project (usedPackages:Map<_,_>) gitRemoteItems opti
             |> Seq.filter (fun (_,_,contentCopySettings,_) -> contentCopySettings <> ContentCopySettings.Omit)
             |> Seq.map (fun ((group, packName),v,s,s') -> s,s',findPackageFolder root (group, packName) v)
             |> Seq.choose (fun (contentCopySettings,contentCopyToOutputSettings,packageDir) ->
+                printfn  "%s" packageDir.FullName
                 packageDir.GetDirectories "Content"
                 |> Array.append (packageDir.GetDirectories "content")
                 |> Array.tryFind (fun _ -> true)
                 |> Option.map (fun x -> x,contentCopySettings,contentCopyToOutputSettings))
             |> Seq.toList
 
+        printfn "%A" packageDirectoriesWithContent
```

Comment:
Cause of https://github.com/fsprojects/Paket/issues/2746

Reply:
OMG. Can you please remove it and republish. Looks like I'm a bit blind after parental leave